internal/playlist_creator/TagReader.go

```diff
@@ -1,13 +1,15 @@
 package playlist_creator
 
 import (
+	"fmt"
 	"github.com/bogem/id3v2/v2"
 )
 
 func createPlaylistEntryName(fileEntry FileEntry) string {
 
 	tag, err := id3v2.Open(fileEntry.Path+"/"+fileEntry.FileName, id3v2.Options{Parse: true})
 	if err != nil {
+		fmt.Printf("Warning: unable to get taf from file: %s", err.Error())
```

Comment:
it's a good idea to print warning logs.
but i'd be consistent with either picking `log.Printf` or `fmt.Printf`. (i see you've used `log.Printf` for `readFilesRecursively`.)

and `tag` 🙃
```suggestion
		log.Printf("Warning: unable to get tag from file: %s", err.Error())
```